### ISSUE TITLE
Publishing new release

### ISSUE THREAD

**Issue by elichai** [eqrion/cbindgen#290]
Hey, If you could publish a new release soon it would be great, right now I'm using this for non default imports: 

```
fn add_header<P: AsRef<Path>>(file_path: P, header: &[u8]) {
    let file_path = file_path.as_ref();
    let mut original = File::open(file_path.clone()).unwrap();
    let mut temp = NamedTempFile::new_in(".").unwrap();
    temp.write_all(header).unwrap();
    temp.write(b"\n").unwrap();
    io::copy(&mut original, &mut temp).unwrap();
    drop(original);
    temp.persist(file_path).unwrap();
}
```

**Comment by eqrion** [comment]
I'd like to do a release with #271 in it, so I'll try and give that a day or so to be resolved. If that's continued to be blocked, we can do a release for this fix.

**Comment by paulrouget** [comment]
#271 has merged. Can you publish the new release?

**Comment by eqrion** [comment]
I've published [0.8.0](https://github.com/eqrion/cbindgen/commit/e712cc42c759ace3e47a6ee9fdbff8c4f337cec1).
